### ISSUE TITLE
Install Instructions

### ISSUE THREAD

**Issue by ChristopherEeles** [svmiller/svm-r-markdown-templates#22]
Would you be able to add installation instructions for these templates to your README.md file? I am not familiar with installing .Rmd templates manually.

**Comment by svmiller** [comment]
Hi Christopher,

You don't need to "install" it, per se. Just download the particular `.tex` template to somewhere on your directory and, in the YAML of your `.Rmd` file, point to the relative path of the template. So, if you download `svm-latex-ms.tex` to your Desktop directory on a Mac (e.g. `/Users/ChristopherEeles/Desktop/svm-latex-ms.tex`, make the `template:` field in the YAML to be template: `~/Desktop/svm-latex-ms.tex`.

**Comment by useR3906** [comment]
I tried as you suggested but I get an odd error:
`"template" (line 882, column 110):
unexpected "<"
expecting letter
Error: pandoc document conversion failed with error 5
Execution halted`

Can you please check if it is an issue with the file? (I am using R 4.0.0)

**Comment by svmiller** [comment]
That `"<"` tells me you copy-pasted it or saved it in a way that also ported some HTML tags with it. Check the .tex file and see what's inside it..

**Comment by hugomachado8** [comment]
Hi! Thank you for you work with this, it's very helpful. I have downloaded your R package, but I need the template for the statement, and it didn't come with it. I've tried saving the .tex code for the template statement (`svm-latex-statement`) in the template directory, then running the markdown code you provided, but pandoc is having issues finding the file (even though I have specified the path correctly). Any suggestions on what I should to get this template working? The ones that work have a dedicated folder with the `template.tex` file in `resources`, a `template.YAML` file in the main folder and the `Skeleton.Rmd` file in the `Skeleton` folder, maybe it has something to do with that?
